src/map/prepareLayer.js

```diff
@@ -279,10 +279,16 @@ function fetchMultipleSources(mapId, layer, dispatch) {
           // Add unique "one"s to mergedData
           prevData[datum[joinProp]] = { ...datum };
           prevData[datum[joinProp]][(relation['many-prop'] || 'many')] = [];
+          if (customFilter) {
+            prevData[datum[joinProp]].no_of_reports = '0';
```

Comment:
If we add another mapspec property here we could reference that instead of `no_of_reports`, maybe something like `customFilterField` or something. OR change `hasCustomFilter` to a string which is used here instead of `no_of_reports`.

```
layerObj.aggregate.hasCustomFilter = true;
layerObj.aggregate.customFilterField = 'No of Reports';

prevData[datum[joinProp]][layerObj.aggregate.customFilterField] = '0';
```
OR
```
layerObj.aggregate.hasCustomFilter = 'No of Reports';

prevData[datum[joinProp]][layerObj.aggregate.hasCustomFilter] = '0';
```

I think the second sounds most straightforward.